common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -51,6 +51,10 @@ class WhiskConfig(requiredProperties: Map[String, String],
     properties
   }
 
+  def getAsBoolean(key: String, defaultValaue :Boolean) :Boolean = {
+    Try(getProperty(key).toBoolean).getOrElse(defaultValaue)
+  }
```

Comment:
Should live in Config.scala (there are a few other of those methods already)

Reply:
Agree - seems like base class functionality.